modules/ztp-provisioning-lvm-storage.adoc

```diff
@@ -37,6 +39,24 @@ You use {lvms} to persist event subscriptions when you use PTP events or bare-me
     channel: stable-{product-version}
   policyName: subscription-policies
 ----
++
+[NOTE]
+====
+The Storage LVMO subscription is deprecated. In future releases of {product-title}, the storage LVMO subscription will not be available. Instead, you must use the Storage LVMS subscription. 
+
+In {product-title} {product-version}, you can use the Storage LVMS subscription. This subscription does not require manual overrides in the `common-ranGen.yaml` file. Add the following YAML to `spec.sourceFiles` in the `common-ranGen.yaml` file to use the Storage LVMS subscription:
```

Comment:
I would say: you can use the Storage LVMS subscription instead of the local storage subscription. I mean, I do not see any use case where a SNO requires two storage solutions (LSO and LVMS).

Reply:
Updated. 

Alberto approved this PR on Slack.